Airports/EGPG/SMR/regions.txt

```diff
@@ -0,0 +1,171 @@
+N055.58.30.458 W003.58.11.725
+N055.58.30.415 W003.58.11.722
+N055.58.29.592 W003.58.11.117
+N055.58.29.615 W003.58.10.995
+N055.58.29.442 W003.58.10.784
+N055.58.29.269 W003.58.11.060
+N055.58.28.075 W003.58.10.388
+N055.58.28.284 W003.58.09.380
+N055.58.28.981 W003.58.09.763
+N055.58.29.149 W003.58.08.591
+N055.58.30.474 W003.58.09.251
+N055.58.30.702 W003.58.09.201
+N055.58.30.920 W003.58.09.233
+N055.58.30.972 W003.58.09.176
+N055.58.31.177 W003.58.09.298
+N055.58.32.253 W003.58.09.080
+N055.58.32.134 W003.58.09.745
+; - Dissued
+REGIONNAME Cumbernauld
+smrPGdissued N055.58.30.372 W003.58.09.199
+N055.58.30.437 W003.58.08.848
+N055.58.30.451 W003.58.08.599
+N055.58.30.613 W003.58.07.647
+N055.58.30.762 W003.58.07.740
+N055.58.30.610 W003.58.08.639
+N055.58.30.529 W003.58.08.891
+N055.58.30.473 W003.58.09.248
+; - Taxiway+parking
+REGIONNAME Cumbernauld
+smrPGTaxiway N055.58.30.587 W003.58.18.942
+N055.58.30.570 W003.58.18.880
+N055.58.30.547 W003.58.18.839
+N055.58.30.515 W003.58.18.798
+N055.58.30.441 W003.58.18.742
+N055.58.29.352 W003.58.18.163
+N055.58.29.279 W003.58.18.137
+N055.58.29.239 W003.58.18.132
+N055.58.29.201 W003.58.18.145
+N055.58.29.163 W003.58.18.163
+N055.58.29.131 W003.58.18.201
+N055.58.29.102 W003.58.18.246
+N055.58.28.947 W003.58.19.111
+N055.58.28.016 W003.58.18.596
+N055.58.29.037 W003.58.12.877
+N055.58.29.949 W003.58.13.374
+N055.58.29.283 W003.58.17.195
+N055.58.29.272 W003.58.17.304
+N055.58.29.300 W003.58.17.440
+N055.58.29.344 W003.58.17.521
+N055.58.29.398 W003.58.17.575
+N055.58.30.422 W003.58.18.100
+N055.58.30.577 W003.58.18.168
+N055.58.30.667 W003.58.18.155
+N055.58.30.747 W003.58.18.082
+N055.58.30.786 W003.58.18.023
+N055.58.30.602 W003.58.19.075
+N055.58.30.594 W003.58.18.986
+; - Taxiway+parking
+REGIONNAME Cumbernauld
+6383724 N055.58.29.547 W003.58.24.902
+N055.58.29.527 W003.58.24.831
+N055.58.29.509 W003.58.24.789
+N055.58.29.493 W003.58.24.760
+N055.58.29.469 W003.58.24.726
+N055.58.29.444 W003.58.24.704
+N055.58.28.207 W003.58.24.049
+N055.58.28.167 W003.58.24.047
+N055.58.28.144 W003.58.24.062
+N055.58.28.102 W003.58.24.098
+N055.58.28.069 W003.58.24.149
+N055.58.28.049 W003.58.24.197
+N055.58.28.031 W003.58.24.247
+N055.58.28.019 W003.58.24.307
+N055.58.27.824 W003.58.25.464
+N055.58.26.910 W003.58.24.947
+N055.58.27.772 W003.58.20.071
+N055.58.28.215 W003.58.20.075
+N055.58.28.439 W003.58.18.811
+N055.58.28.727 W003.58.18.974
+N055.58.28.508 W003.58.20.212
+N055.58.28.714 W003.58.20.288
+N055.58.28.282 W003.58.22.807
+N055.58.28.232 W003.58.23.099
+N055.58.28.222 W003.58.23.240
+N055.58.28.241 W003.58.23.342
+N055.58.28.268 W003.58.23.419
+N055.58.28.300 W003.58.23.470
+N055.58.28.324 W003.58.23.491
+N055.58.29.477 W003.58.24.133
+N055.58.29.574 W003.58.24.167
+N055.58.29.632 W003.58.24.150
+N055.58.29.699 W003.58.24.075
+N055.58.29.735 W003.58.24.014
+N055.58.29.555 W003.58.25.048
+N055.58.29.556 W003.58.24.967
+; - Buildings
+REGIONNAME Cumbernauld
+1973790 N055.58.27.020 W003.58.18.029
```

Comment:
Same here, and for all the other colours.  Please define them and then rename them here